app/assets/stylesheets/projects.scss

```diff
@@ -0,0 +1,8 @@
+.visit-links p {
+  a {
+    color: #fff !important;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used